src/js/containers/search/visualizations/rank/RankVisualizationWrapperContainer.jsx

```diff
@@ -160,12 +162,14 @@ export class RankVisualizationWrapperContainer extends React.Component {
                     return;
                 }
 
+                const responseDetail = get(err, 'response.data.detail', '');
                 this.props.setAppliedFilterCompletion(true);
                 this.apiRequest = null;
                 console.log(err);
                 this.setState({
                     loading: false,
-                    error: true
+                    error: true,
+                    recipientError: responseDetail === 'Current filters return too many unique items. Narrow filters to return results.'
```

Comment:
This is the specific error from the API when you're paging beyond 10K results?

Reply:
Yeh. With the new elasticsearch implementation, you cannot go past 10k unique records, which is done easily with spending by recipient. Since we only want to catch this with the Elasticsearch implementation I decided to check for the specific error for the time being.